### PR TITLE
Feat: show all screenshots in carousel

### LINKED ISSUE #71: Show all screenshots in the frontend
Look at js for an example

carousel to display all images and on click show the diff screen sizes

### PR DESCRIPTION
**Pull request by eosza**
Fixes #71 
In this PR:

- Adds pure-react-carousel to package.-lock.json using:  ``` npm i pure-react-carousel ```

- Adds a carousel to display all screenshots if there is more than one screenshot

- Default behavior: show full page screenshot if there isn't more than one screenshot

- Adds full page screenshot button to carousel. On press the button shows the full page screenshot in a modal

Note the carousel itself fills but the part of the page that it's been allocated. This is a layout issue with the other components that needs to be discussed.

### CODE REVIEW

File: src/views/tickets/Screenshot.js [hunk 1]
```diff
@@ -1,18 +1,63 @@
...
+    );
+    this.carousel.push(image);
+
     try {
-      let value = await getTicket(this.props.ticketID);
+      let ticket = await getTicket(this.props.ticketID);
+      const length =
+        this.state.carouselLength + ticket.ticket.screenshots.length;
+      if (length > 1) {
+        this.setState({ singleScreenshot: false });
```
Comment (williamtdr) [inline src/views/tickets/Screenshot.js]: Could we use `this.state.carouselLength` in place of this extra variable? I might be missing something here.

Comment (eosza) [reply src/views/tickets/Screenshot.js]: hahaha yes let me fix that, I'm not used there being a state.

Comment (williamtdr) [reply src/views/tickets/Screenshot.js]: Thanks! I was thinking more about `singleScreenshot` though - do we need that where it's used, or can we check `carouselLength > 1`?

Comment (eosza) [reply src/views/tickets/Screenshot.js]: I'll fix that too!

**Pull request merged**